crates/viewer/re_viewer/src/app.rs

```diff
@@ -171,6 +171,8 @@ struct PendingFilePromise {
     promise: poll_promise::Promise<Vec<re_data_source::FileContents>>,
 }
 
+type ReceiveSetTable = parking_lot::Mutex<Vec<crossbeam::channel::Receiver<TableMsg>>>;
```

Comment:
`ReceiveSetTable` reads weird to me

```suggestion
type TableReceiverSet = parking_lot::Mutex<Vec<crossbeam::channel::Receiver<TableMsg>>>;
```

Reply:
I named it this way to make it somewhat symmetric with `ReceiveSet<LogMsg>`.